make_html.py

```diff
@@ -312,6 +315,10 @@ def get_data():
                 f_data.dependencies = [
                     data[k] for k in nx.ancestors(graph, f_import) if k in data
                 ]
+
+                # Save topological order into data.
+                f_data.topol_order = topol_order[f_import] # graph.nodes[f_import]["topol_order"]
```

Comment:
There's a `dataclass` higher up in the file which should have a new field added to document this